nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/commands/IndexingCommandsStacker.java

```diff
@@ -70,9 +72,22 @@ public void stackCommand(DocumentEventContext docCtx, String eventId) {
             return;
         }
         if ("/".equals(doc.getPathAsString())) {
-            log.debug("Skip indexing command for root document");
-            return;
+            if (log.isDebugEnabled()) {
+                log.debug("Skip indexing command for root document");
+            }
+            if (eventId.equals(DOCUMENT_SECURITY_UPDATED)) {
```

Comment:
You could have done the `.equals` around the other way to guard against null pointer.

Reply:
FindBugs taught me that!